zebra-state/src/request.rs

```diff
@@ -849,6 +849,36 @@ pub enum ReadRequest {
     /// * [`ReadResponse::OrchardTree(None)`](crate::ReadResponse::OrchardTree) otherwise.
     OrchardTree(HashOrHeight),
 
+    /// Returns a list of Sapling note commitment subtrees by their indexes,
+    /// starting at `start_index`, and returning up to `limit` subtrees.
+    ///
+    /// Returns
+    ///
+    /// * [`ReadResponse::SaplingSubtree(BTreeMap<_, NoteCommitmentSubtreeData<_>>))`](crate::ReadResponse::SaplingSubtrees)
+    ///
+    /// If there is no subtree at `start_index`, returns an empty list.
```

Comment:
###### Optional

We can make this more consistent with the rest of the requests:
```suggestion
    /// * An empty list if there is no subtree at `start_index`
    ///
```

Reply:
I think these are good suggestions, and it seems like you have a good idea how you'd like the documentation to look. Would you mind submitting a PR with these changes?

Or are you happy for me to accept all your suggestions and leave it at that?

Reply:
I'll submit a PR.